src/sentry/api/endpoints/organization_member/team_details.py

```diff
@@ -83,8 +93,10 @@ def _is_org_owner_or_manager(access: Access) -> bool:
     return any("org:write" in role.scopes for role in roles)
 
 
+@extend_schema(tags=["Teams"])
 @region_silo_endpoint
 class OrganizationMemberTeamDetailsEndpoint(OrganizationMemberEndpoint):
+    public = {"DELETE", "POST"}
```

Comment:
just for my own sake -- how are we determining what is public? what is currently public?